athena-dynamodb/README.md

```diff
@@ -20,6 +20,8 @@ generation for a stronger source of encryption keys. (e.g. a7e63k4b-8loc-40db-a2
 Setting this to false will disable spill encryption. You may wish to disable this for improved performance, especially if your spill location in S3 uses S3 Server Side Encryption. (e.g. True or False)
 6. **disable_glue** - (Optional) If present, with any value except false, the connector will no longer attempt to retrieve supplemental metadata from Glue.
 7. **glue_catalog** - (Optional) Can be used to target a cross-account Glue catalog. By default the connector will attempt to get metadata from its own Glue account.
+8. **DISABLE_PROJECTION_AND_CASING_ENV** - (Optional) Can be used to disable projection and casing in order to be able to query DynamoDB tables with casing in their column names without having to specify a "columnMapping" property on their Glue table. Go here for more info [DISABLE_PROJECTION_AND_CASING_ENV](#DISABLE_PROJECTION_AND_CASING_ENV).
```

Comment:
I am not sure we should expose this option. This was a bug, it should just get fixed. If someone wants the old behavior (instead of just fixing your glue definitions) then they can use an old version of the connector. These connectors have a problem with already being too difficult to use, this just adds to that mental strain.